pkgs/desktops/gnome-3/core/gdm/default.nix

```diff
@@ -59,7 +59,13 @@ stdenv.mkDerivation rec {
     echo #!/bin/sh > build-aux/find-x-server.sh
     echo "echo ${stdenv.lib.getBin xorg.xorgserver}/bin/X" >> build-aux/find-x-server.sh
     patchShebangs build-aux/find-x-server.sh
-  '';
+  '' +
+  # fix a typo in data/meson.build
+  ''
+    substituteInPlace data/meson.build \
```

Comment:
if you could please drop this and apply https://gitlab.gnome.org/GNOME/gdm/-/commit/a798460abcba6e3257bf83441dd2b5aa53491646

Reply:
It won't apply:

```
applying patch /nix/store/ch0748yx0lm6q8a05nsqirw7iwqm729m-a798460abcba6e3257bf83441dd2b5aa53491646.diff
patching file data/meson.build
Reversed (or previously applied) patch detected!  Assume -R? [n]
Apply anyway? [n]
```

No idea why...

Reply:
Looks like the patch is already in 3.38.0.